docs/clients/promtail/stages/template.md

```diff
@@ -58,6 +59,32 @@ converts its value to be all lowercase. For example, if the extracted map
 contained `app` with a value of `LOKI`, this pipeline would change its value to
 `loki`.
 
+```yaml
+- template:
+    source: output_msg
+    template: '{{ .level }} for app {{ ToUpper .app }}'
```

Comment:
Does that work shouldn't it be ` '{{ .level }} for app {{  .app | ToUpper }}'` ?

Reply:
same thing line 78

Reply:
Both will work. Other examples in doc uses this format `ToLower .Value`. Kept the same for uniformity.